sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/display/DisplayDataMatchers.java

```diff
@@ -57,6 +57,14 @@ private DisplayDataMatchers() {}
   }
 
   /**
+   * Creates a matcher that matches if the examined {@link DisplayData} contains an item with the
+   * specified key.
+   */
+  public static Matcher<DisplayData> hasDisplayItem(String key) {
```

Comment:
This one's a bit funky because `hasDisplayItem(..)` is specifying some assertions on a `DisplayData.Item`, but returns a `DisplayData` Matcher. There are other APIs that return `DisplayData.Item` matchers. It's a shortcut syntax for a common pattern: `hasDisplayItem(String) == hasDisplayItem(hasKey(String))`.

I'm going to leave as-is because changing to `hasKey` would cause type strangeness with other `DisplayData.Item` matcher APIs.